src/executables/util_sa.c

```diff
@@ -2911,8 +2913,11 @@ synccoll_check (const char *db_name, int *db_obs_coll_cnt, int *new_sys_coll_cnt
 		  fprintf (stdout, "%s\n", db_get_string (&class_name));
 
 		  /* output query to fix schema */
-		  snprintf (query, sizeof (query) - 1, "ALTER TABLE [%s] " "COLLATE utf8_bin;",
-			    db_get_string (&class_name));
+		  strcpy (split_owner_name, db_get_string (&class_name));
+		  split_name = strchr (split_owner_name, '.');
+		  *split_name++ = '\0';
```

Comment:
If the `split_owner_name` array is initialized as 0
```suggestion
		  *split_name++ = '\0';
```

Reply:
Thank you. I will fix it.

Reply:
Done (0a529c1)